docs/reference/aggregations/bucket/daterange-aggregation.asciidoc

```diff
@@ -111,3 +111,35 @@ Zone:: 'Z' outputs offset without a colon, 'ZZ' outputs the offset with a colon,
 Zone names:: Time zone names ('z') cannot be parsed.
 
 Any characters in the pattern that are not in the ranges of ['a'..'z'] and ['A'..'Z'] will be treated as quoted text. For instance, characters like ':', '.', ' ', '#' and '?' will appear in the resulting time text even they are not embraced within single quotes.
+
+[[time-zones]]
+==== Time zone in date range aggregations
+
+Dates can be converted from another time zone to UTC by specifying the `time_zone` parameter.
+
+Time zones may either be specified as an ISO 8601 UTC offset (e.g. +01:00 or -08:00) or as one of
+the the http://joda-time.sourceforge.net/timezones.html[time zone ids] from the TZ database.
```

Comment:
Actually, the notes on that page say those are the time zones as of 2013d.  Joda current version 2.9.2 ships with tzdb 2016a - so there are a few time zones that have been added since then that aren't in the list.

Speaking of which, Wikipedia is at 2015g, I need to go update that to 2016b, I'll go do that now. :)

@jodastephen - Do you intend to keep the list of tzdb zones on joda.org updated? Or should a more neutral source like Wikipedia be prefered?  Also - are there any zones added that are not tzdb standard?  Thanks.